translations/fr.json

```diff
@@ -0,0 +1,19 @@
+{
+    "Event Details": "Détails de l'évènement",
```

Comment:
@bpvcode can we match these keys to be the same as the german ones? I need to check the Spanish ones, but ideally, all of these will be consistent across all the languages so that the token is universal and just searches for the right language. So if you can match these keys to be 
`eventDetaills` instead of `Event Details`, etc that would be awesome!!

Reply:
Oh sorry about this, my mistake... I was clearly distracted.
I will do the changes right now :)
Thanks for the feedback